src/code-generators/typescript-generators/namedImport.ts

```diff
@@ -53,26 +57,54 @@ export function generateNamedImport(
     const lib = `${stringQuoteStyle}${imp.libraryName}${stringQuoteStyle}${eol}`;
 
     const specifiers = imp.specifiers.sort(specifierSort).map(o => generateSymbolSpecifier(o)).join(', ');
-    let importSpecifiers = `${space}${specifiers}${space}`;
-    if (importSpecifiers.trim().length === 0) {
-        importSpecifiers = ' ';
-    }
-
-    const importString = importTemplate(
-        getImportSpecifiers(imp, spaceBraces),
-        lib,
-    );
+    let retVal:string = '';
```

Comment:
missing space in code formatting

Reply:
And please use the whole name `returnValue` or `generatedImport` or something like that. I'm not a big fan of abbreviations since we're not coding C or something ;-)

Reply:
yup.